CS191/posts/static/js/grid.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+
+	// Affix behavior for sidebar
+	$('#topics').affix({
+		offset: {
+			top: ($('#topics').offset().top) - 20,
+			bottom: ($('#topics').offset().bottom)
+		}
+	});
+
+	$('[data-clampedwidth]').each(function () {
```

Comment:
Maybe comment whatever this is. It isn't super confusing, but just so we can glance at it and know immediately